code/modules/clothing/clothing.dm

```diff
@@ -160,7 +160,7 @@ var/list/damaged_clothes_icons = list()
 		add_overlay(damaged_clothes_icon, 1)
 	else
 		damaged_clothes = 0
-		cut_overlay(damaged_clothes_icons[index], TRUE)

```

Comment:
I haven't touched this line so I think this might be sublime messing with line breaks or something, similar to another PR